src/config.hpp

```diff
@@ -50,4 +53,30 @@ const auto dxp_screenshot_period = std::chrono::seconds (1);
 ///
 /// Otherwise leave empty.
 ///
-const std::vector<uint32_t> dxp_viewport = {};
+const std::vector<uint32_t> dxp_viewport = { 0, 0, 0, 0, 0, 0, 0, 0 };
+
+///
+/// Key settings:
+///
+/// Next - preselect next desktop
+/// (Right & down arrows by default)
+///
+/// Prev - preselect previous desktop
+/// (Left & up arrows by default)
+///
+/// Exit - kill dxp
+/// (Escape by default)
+///
+/// Choose - select desktop
+/// (Enter by deafault)
+///
+/// View keynames.hpp for a list of supported keys
+/// and their names
+
+keys_for action_next = { Down, Right, latin_l, latin_k };
+
+keys_for action_prev = { Left, Up, latin_h, latin_j };
```

Comment:
Swap j and k buttons. j is next and k is prev

Reply:
will do